components/forms/RadioGroup/react.js

```diff
@@ -0,0 +1,42 @@
+import React, { memo } from 'react'
+import PropTypes from 'prop-types'
+import { React as Radio } from 'components/forms/RadioButton'
+import s from './style.module.scss'
+
+const RadioGroup = ({
+  checkedChoice, choices, direction, onChange,
+}) => {
+  const radios = choices.map(({ label, value }) => (
+    <li key={value} className={direction === 'column' ? '' : s['radio-group__row']}>
+      <Radio
+        checked={checkedChoice === value}
+        onClick={event => onChange(event.target.value)}
+        onChange={event => onChange(event.target.value)}
+        label={label}
+        value={value}
+      />
+    </li>
+  ))
+  return <ul className={s['radio-group']}>{radios}</ul>
```

Comment:
Is there a browser default margin/padding from the `<ul>`? I figure this would be zero'd out. Haven't tested in Storybook